spec/spec_helper.rb

```diff
@@ -3,6 +3,11 @@
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
 require 'capybara/rspec'
+require 'capybara/email/rspec'
+
+Capybara.app_host = "http://localhost:3000"
+Capybara.server_host = "localhost"
+Capybara.server_port = "3000"
```

Comment:
has the same reason as stated above... was just a trial to get rid of the "examle.com" used by capybara. Refer to the same post: https://www.gotealeaf.com/lessons/5516459d/home#post_15302